### PR TITLE
Add support for Akash Network Token (AKT)

### PR DESCRIPTION
**Pull request by sunnya97**

### CODE REVIEW

File: rotkehlchen/data/all_assets.json [hunk 1]
```diff
@@ -414,6 +414,15 @@
         "symbol": "AKRO",
         "type": "ethereum token"
     },
+    "AKT": {
+        "coingecko": "akash-network",
+        "cryptocompare": "akt",
+        "ethereum_token_decimals": 18,
```
Comment (LefterisJP) [inline rotkehlchen/data/all_assets.json:420]: this key is not needed for non ethereum tokens

**Review by LefterisJP — COMMENTED** [review]
Hey @sunnya97 thanks for the PR. Can you also do the small change requested and add the hash to the md5 test?

https://github.com/rotki/rotki/blob/01a3f72feff69f7be39e4060b946d8b1164d5c02/rotkehlchen/tests/unit/test_assets.py#L195

It's really strange the test did not fail by the way. I have to look it up.

**Comment by LefterisJP** [comment]
I want to add 2 more tokens right now, so I will just take over the PR to add them in one go and increase version once.

**Review by LefterisJP — APPROVED** [review]
LGTM

**Comment by sunnya97** [comment]
Ah gotcha, missed the part of the instructions for "And the version in the meta file should also be bumped".  Thanks for covering it for me!

**Pull request merged**